queries.sql

```diff
@@ -14,4 +14,79 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg  >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg  >= 10.4 AND weight_kg <= 17.3;
+
+/* Part 1 */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+/* Part 2 */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
```

Comment:
- [ ] you should verify if the changes were made before and after commit as well

Reply:
<img width="576" alt="2" src="https://user-images.githubusercontent.com/53530780/189984371-e6a150a7-66a5-41b7-8db1-2edc60d3d596.png">